lib/src/base-styles/_nx-lists.scss

```diff
@@ -133,104 +130,64 @@
   }
 
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color-light;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
+      background-color: #d3e6ff;
       pointer-events: none;
     }
 
-    &:not(.nx-list__item--empty):not(.nx-disabled):not([disabled]) {
+    &:not(.nx-list__item--empty):not(.disabled):not([disabled]),
+    &:not(.nx-list__item--error) {
       &:hover, &:focus {
-        background-color: $nx-color-blue-96;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
-        color: $nx-text-color;
+        background-color: #f1f3fa;
         cursor: pointer;
         outline-style: none; // removes :focus outline
 
         .fa-chevron-right {
-          color: $nx-color-blue-41;
+          color: #045dd1;
         }
       }
 
       &:active {
-        background-color: $nx-grey-1;
-        border-top: $nx-border-row-hover;
-        border-bottom: $nx-border-row-hover;
+        background-color: #d3e6ff;
```

Comment:
The active style isn't explicitly shown in zeplin

Reply:
> The active style isn't explicitly shown in zeplin

I assumed that "active" and "clicked" were the same. @dturner32 ?

Reply:
> > The active style isn't explicitly shown in zeplin
> 
> I assumed that "active" and "clicked" were the same. @dturner32 ?

that is correct